### DISCUSSION TITLE
semantic release tool shows the last tag is v1.0.0 for branch alpha

### DISCUSSION THREAD

**Discussion by omar-khalid1** [semantic-release/semantic-release#3524] [General]
semantic release tool shows the last tag is v1.0.0 for branch alpha however , I created alpha branch from beta branch which had the latest release of v2.0.0-beta.1 , so whenever i create a major release , it increment to v2.0.0-alpha.1 , do i miss anything?

**Comment by travi** [comment]
Have you force pushed since the beta version was released? Rewriting history will cause previous tags to no longer be associated with the commits in the history of your branch
